### PR TITLE
Dep update aurelia-rc1

### PR DESCRIPTION
**Pull request by doktordirk**
- dependencies update to aurelia rc1
- build task copied from aurelia and adapted

### CODE REVIEW

**Comment by doktordirk** [comment]
not working yet for aurelia-cli (pathes of jwr-decode not resolved correctly) and new webpack-plugin (wierd DI issue)

File: build/tasks/build.js [hunk 1]
```diff
@@ -3,125 +3,158 @@ var runSequence = require('run-sequence');
...
+
+compileToModules.forEach(function(moduleType){
+  gulp.task('build-babel-' + moduleType, function () {
+    return srcForBabel()
+      .pipe(to5(assign({}, compilerOptions[moduleType]())))
+      .pipe(cleanGeneratedCode())
+      .pipe(gulp.dest(paths.output + moduleType));
+  });
+
+  if (moduleType === 'native-modules') return; // typescript doesn't support the combination of: es5 + native modules
```
Comment (icanhasjonas) [inline build/tasks/build.js:90]: It's a recently added feature in TypeScript 2.0 (currently in beta)
https://github.com/Microsoft/TypeScript/issues/6319

Comment (doktordirk) [inline build/tasks/build.js:90]: oh, i just copy the build from aurelia. but if you can help with the webpack and cli issues ... :smile:

**Comment by doktordirk** [comment]
ps: atm can be tested with eg `jspm i aurelia-authentication=github:doktordirk/aurelia-authentication@dep-update` or `npm i github:doktordirk/aurelia-authentication#dep-update` (i think)

File: build/tasks/build.js (hunk 1 above)
Comment (RWOverdijk) [inline build/tasks/build.js:90]: Curly brace

Comment (doktordirk) [inline build/tasks/build.js:90]: that's just copied from aurelia. seems easier not change as few as possible to make updating easier

**Comment by RWOverdijk** [comment]
Looks... Okay. I haven't tested it yet but if anyone else could that'd be cool.

**Pull request merged**